staking/programs/staking/src/state/positions.rs

```diff
@@ -827,4 +989,414 @@ pub mod tests {
 
         true
     }
+
+
+    #[quickcheck]
+    fn slash_positions(input: (Vec<Position>, u8, u64)) -> bool {
```

Comment:
cant you do `fn slash_positions(positions: Vec<Position>, index: u8, amount: u64)) -> bool`?

Reply:
done